### PR TITLE
Update : Change color scheme, Preview Layout, Sample input HTML

### PR DESCRIPTION
**Pull request by rayasabari**
**Update**

- Change sky color to pink, gray to slate, and dark mode color
- Edit Preview Component layout and make it responsive
- Edit sample input html component
- Add script for watching css and minify for production

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,8 @@
     "precommit": "npm run lint",
     "lint:watch": "esw src --color --watch",
     "css": "rm public/tailwind.css && npx tailwindcss -i styles/globals.css -o public/tailwind.css --postcss",
+    "watch": "npx tailwindcss -i styles/globals.css -o public/tailwind.css --watch",
+    "prod": "npx tailwindcss -o public/tailwind.css --minify",
```
Comment (Gash21) [inline package.json:14]: just put --minify on css script

**Review by Gash21 — CHANGES_REQUESTED** [review]
please check comment

File: package.json (hunk 1 above)
Comment (rayasabari) [reply package.json]: Done!

**Review by Gash21 — APPROVED** [review]
Cool! Lets deploy!

**Pull request merged**